java/src/com/ibm/streamsx/topology/internal/functional/ops/FunctionSplit.java

```diff
@@ -0,0 +1,60 @@
+/*
+# Licensed Materials - Property of IBM
+# Copyright IBM Corp. 2015  
+ */
+package com.ibm.streamsx.topology.internal.functional.ops;
+
+import static com.ibm.streamsx.topology.internal.functional.FunctionalHelper.getInputMapping;
+import static com.ibm.streamsx.topology.internal.functional.FunctionalHelper.getLogicObject;
+
+import java.util.List;
+
+import com.ibm.streams.operator.OperatorContext;
+import com.ibm.streams.operator.OutputTuple;
+import com.ibm.streams.operator.StreamingInput;
+import com.ibm.streams.operator.StreamingOutput;
+import com.ibm.streams.operator.Tuple;
+import com.ibm.streams.operator.model.Icons;
+import com.ibm.streams.operator.model.InputPortSet;
+import com.ibm.streams.operator.model.OutputPortSet;
+import com.ibm.streams.operator.model.PrimitiveOperator;
+import com.ibm.streamsx.topology.function.ToIntFunction;
+import com.ibm.streamsx.topology.internal.spljava.SPLMapping;
+
+@PrimitiveOperator
+@InputPortSet(cardinality = 1)
+@OutputPortSet(cardinality = -1)
```

Comment:
I do seem to recall some doc that said the default was unlimited, but in fact it is not.
removing the decl yields:

java.lang.IllegalStateException: Operator com.ibm.streamsx.topology.functional.java::FunctionSplit does not allow output ports.